assets/maps/prefabs/prefab_securitron_academy.dmm

```diff
@@ -0,0 +1,224 @@
+"lV" = (/obj/machinery/light/incandescent{dir = 1},/turf/simulated/floor/wood/two,/area/prefab/secbot_academy)
+"mr" = (/obj/machinery/door/airlock/pyro,/turf/simulated/floor/wood/two,/area/prefab/secbot_academy)
+"my" = (/obj/decal/poster/wallsign/poster_nt{pixel_y = 32},/turf/simulated/floor/blueblack{dir = 9},/area/prefab/secbot_academy)
+"mY" = (/obj/decal/tile_edge/stripe{dir = 6; icon_state = "xtra_bigstripe-edge"},/turf/simulated/floor/shuttlebay,/area/noGenerate)
+"nG" = (/obj/deployable_turret/riot{active = 1; anchored = 1; desc = "A Nanotrasen Automatic Riot Control System repurposed for home security."; dir = 4},/turf/simulated/floor/redblack{dir = 8},/area/prefab/secbot_academy)
+"nI" = (/obj/decal/poster/wallsign/cdnp{pixel_y = 32},/turf/simulated/floor/darkblue,/area/prefab/secbot_academy)
+"nJ" = (/obj/machinery/door/airlock/pyro{dir = 4},/turf/simulated/floor/wood/two,/area/prefab/secbot_academy)
+"om" = (/obj/table/wood/auto/desk,/obj/item/paper{pixel_x = 4; pixel_y = 5},/obj/item/paper{pixel_x = -7; pixel_y = 5},/obj/item/paper{pixel_x = -9; pixel_y = -3},/obj/item/paper{pixel_x = 7; pixel_y = -5},/obj/item/paper{pixel_x = -2; pixel_y = -8},/obj/item/pen/fancy,/turf/simulated/floor/wood/two,/area/prefab/secbot_academy)
+"op" = (/obj/machinery/light/incandescent,/turf/simulated/floor/wood,/area/prefab/secbot_academy)
+"oM" = (/obj/decal/poster/wallsign/poster_nt{pixel_x = -32},/turf/simulated/floor/black,/area/prefab/secbot_academy)
+"pi" = (/obj/table/reinforced/bar/auto,/obj/machinery/networked/printer{pixel_y = 3},/turf/simulated/floor/carpet/purple/fancy,/area/prefab/secbot_academy)
+"pY" = (/obj/grille/catwalk{dir = 8},/turf/space,/area/noGenerate)
+"ql" = (/obj/critter/bat/buff,/turf/simulated/floor/redblack{dir = 4},/area/prefab/secbot_academy)
+"qm" = (/obj/decal/tile_edge/stripe{dir = 9; icon_state = "xtra_bigstripe-edge"},/turf/simulated/floor/shuttlebay,/area/noGenerate)
+"qL" = (/obj/table/reinforced/bar/auto,/turf/simulated/floor/carpet/purple/fancy,/area/prefab/secbot_academy)
+"qN" = (/obj/table/auto,/obj/item/medical/medicaldiagnosis/stethoscope{pixel_x = 5},/obj/item/device/light/flashlight/penlight{pixel_x = -6},/turf/simulated/floor/sanitary,/area/prefab/secbot_academy)
+"ru" = (/obj/mic_stand,/turf/simulated/floor/wood/two,/area/prefab/secbot_academy)
+"ss" = (/turf/simulated/floor/redwhite{dir = 6},/area/prefab/secbot_academy)
+"sK" = (/obj/lattice,/turf/space,/area/noGenerate)
+"te" = (/obj/submachine/chef_sink/chem_sink{dir = 1},/turf/simulated/floor/white,/area/prefab/secbot_academy)
+"tC" = (/obj/machinery/recharger,/obj/table/reinforced/industrial/auto,/obj/machinery/light/incandescent,/turf/simulated/floor/redblack,/area/prefab/secbot_academy)
+"tW" = (/obj/item/clothing/head/NTberet,/obj/item/clothing/head/NTberet,/obj/rack,/obj/item/clothing/head/NTberet,/obj/item/clothing/head/NTberet,/turf/simulated/floor/wood/two,/area/prefab/secbot_academy)
```

Comment:
This section ("tW") is giving the Run Linter some grief and is causing a Linter check to fail